.github/workflows/lint-and-test.yml

```diff
@@ -26,3 +26,7 @@ jobs:
       - run: make lint
 
       - run: make coverage
+
+      - run: make install-docs
+
+      - run: make test-docs
```

Comment:
I'd split this in a separate GHA, but required as well in PRs for them to be merged.
This action currently checks the code/functionality follows a standard, but the docs are a different side of this project.
What do you think?

Reply:
Done, I think!